docs/en_US/TrainingService/AdaptDLMode.md

```diff
@@ -0,0 +1,188 @@
+# Run an Experiment on AdaptDL
+
+===
```

Comment:
This `===` does not work here.

Reply:
oh. it is because i saw kubeflow doc also has that https://github.com/microsoft/nni/blob/master/docs/en_US/TrainingService/KubeflowMode.md. thought it was the for some reason. Should I delete this line from adl doc?

Reply:
sure, this line is not desired to appear, seems kubeflow doc has some error, we will fix it later. You could delete this line in adl doc.

Reply:
yup removed.